R/generic-spec.R

```diff
@@ -111,10 +111,16 @@ group_generics <- function() {
     Summary = "Summary",
     Complex = "Complex"
   )
-
+  ## Not yet:
```

Comment:
Is this related to the `%||%` change?

Reply:
It's easiest to keep each PR in a separate branch to avoid that sort of problem. But I can do some surgery on this branch to just include that commit, or probably just do it from first principles since it's such a simple change.